### PR TITLE
Feature/master nat spec format

### PR DESCRIPTION
**Pull request by baizetianxia**
# Description
1 organize code comments according to NatSpec format
2 correct spelling errors
3 add an underscore at the beginning of two local variables

# Ticket

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings

### CODE REVIEW

File: contracts/eth/vault/IETHVault.sol [hunk 1]
```diff
@@ -141,16 +230,12 @@ interface IETHVault {
     /// @notice Get pegToken price in USD(1e18)
```
Comment (mingshengke) [inline contracts/eth/vault/IETHVault.sol:230]: in ETH

File: contracts/eth/vault/IETHVault.sol [hunk 2]
```diff
@@ -159,25 +244,29 @@ interface IETHVault {
...
+    /// @param _minimumAmount The minimum return amount of the ETHi ticket
+    /// @return _sharesAmount The amount of ETHi ticket
     function mint(
         address _asset,
         uint256 _amount,
         uint256 _minimumAmount
     ) external payable returns (uint256 _sharesAmount);
 
-    /// @notice burn ETHi,return stablecoins
+    /// @notice burn ETHi, return ETH
```
Comment (mingshengke) [inline contracts/eth/vault/IETHVault.sol:265]: return  XETH

File: contracts/eth/vault/IETHVault.sol [hunk 3]
```diff
@@ -186,95 +275,84 @@ interface IETHVault {
...
 
     /// @notice Allocate funds in Vault to strategies.
+    /// @param _strategy The specified strategy to lend
+    /// @param _exchangeTokens All exchange info
     function lend(address _strategy, IExchangeAggregator.ExchangeToken[] calldata _exchangeTokens)
         external;
 
     /// @notice Withdraw the funds from specified strategy.
+    /// @param _strategy The specified strategy to redeem
+    /// @param _amount The amount to redeem in USD
```
Comment (mingshengke) [inline contracts/eth/vault/IETHVault.sol:285]: in ETH

**Review by mingshengke — APPROVED** [review]

File: contracts/eth/mock/MockETHVault.sol [hunk 4]
```diff
@@ -58,6 +76,9 @@ contract MockETHVault is AccessControlMixin {
     }
 
     /// @notice Withdraw the funds from specified strategy.
+    /// @param _strategy The specified strategy to redeem
+    /// @param _usdValue The amount to redeem in USD
```
[the remark this answers is not in the document]
Comment (baizetianxia) [reply contracts/eth/mock/MockETHVault.sol:80]:
```
/// @param _usdValue The amount to redeem in USD
```
这个入参_usdValue 要不要改成 _ethValue

**Review by oldJ207 — APPROVED** [review]

File: contracts/eth/strategies/ETHBaseStrategy.sol [hunk 5]
```diff
@@ -14,40 +14,40 @@ import "../oracle/IPriceOracleConsumer.sol";
...
+    /// @inheritdoc IETHStrategy
+    uint16 public override protocol;
 
-    event Repay(
-        uint256 _withdrawShares,
-        uint256 _totalShares,
-        address[] _assets,
-        uint256[] _amounts
-    );
-
```
Comment (ZhangBrian) [inline contracts/eth/strategies/ETHBaseStrategy.sol:41]: Will this modification cause the slot order to change?

**Review by ZhangBrian — APPROVED** [review]

**Review by HHankCCui — APPROVED** [review]

**Review by walkaroundby — APPROVED** [review]

File: contracts/eth/strategies/ETHBaseStrategy.sol (hunk 5 above)
Comment (baizetianxia) [reply contracts/eth/strategies/ETHBaseStrategy.sol:41]: Nope

**Pull request merged**